src/main/java/net/minecraftforge/event/entity/player/AdvancementEvent.java

```diff
@@ -32,6 +34,7 @@
  * <br>
  * This event is fired on the {@link MinecraftForge#EVENT_BUS}.
  */
+@Deprecated()//todo : remove and use AdvancementEvents instead in 1.19
```

Comment:
This class should not be deprecated for removal, and the events classes in `AdvancementEvents` should be moved to be nested/inner classes of this class instead.